torch/_functorch/_aot_autograd/utils.py

```diff
@@ -86,7 +86,7 @@ def _get_autocast_states():
 
 def make_boxed_func(f):
     def g(args):
-        return f(*args)
```

Comment:
Hmm why do you need this? (My guess is that there's a place we are calling `make_boxed_func` where you just want to remove it, because both the inner and outer fn already have boxed calling convention)